sandbox/src/mocks/stu3/retrieveAdviceAndGuidanceConversation/AttachmentPresentInEachDirection.json

```diff
@@ -169,7 +169,7 @@
       "fullUrl": "_baseUrl_/STU3/PractitionerRole/103264994981.R69.SERVICE-PROVIDER-CLINICIAN",
       "resource": {
         "id": "103264994981.R69.SERVICE-PROVIDER-CLINICIAN",
-        "meta": {
+      "meta": {
```

Comment:
Looks like the formatting has gone a little bit weird here, I think `meta` should still be indented to be inline with the `id` tag above.